.gitignore

```diff
@@ -59,3 +59,4 @@ typings/
 
 # CoffeeScript build files
 lib/*.js
+ofirehose.json
```

Comment:
Should probably get rid of this, assuming you agree with the Yargs comment I left below